src/js/view/view.js

```diff
@@ -723,14 +723,18 @@ function View(_api, _model) {
         replaceClass(_playerElement, /jw-state-\S+/, 'jw-state-' + state);
 
         switch (state) {
-            case STATE_IDLE:
             case STATE_ERROR:
-            case STATE_COMPLETE:
                 _this.stopFloating();
                 if (_captionsRenderer) {
                     _captionsRenderer.hide();
                 }
                 break;
+            case STATE_IDLE:
+            case STATE_COMPLETE:
+                if (_captionsRenderer) {
```

Comment:
really not crazy about the duplicate logic but the linter yells if there's not a break statement in the first `if STATE_ERROR` case

Reply:
I would prefer an `if (state === STATE_ERROR) {  _this.stopFloating(); }` block to these duplicate cases.

Reply:
updating!

Reply:
Oh now I see. The other way to do it is put STATE_ERROR first and add a `/* falls through */` comment where the break would be. Ex:

https://github.com/jwplayer/jwplayer/blob/ab663ef80aacba02d295b784ecbdb93461a0ff4c/src/js/program/program-listeners.js#L98

Reply:
Yeah, our linter doesn't allow two of the same case in a single switch though so we couldn't do that since the fallthrough would _also_ need to catch `STATE_ERROR`. that was my first fix but got a big red ❌